stdlib/universe/window.go

```diff
@@ -181,8 +181,11 @@ func createWindowTransformation(id execute.DatasetID, mode execute.AccumulationM
 	cache := execute.NewTableBuilderCache(a.Allocator())
 	d := execute.NewDataset(id, mode, cache)
 
-	bounds := a.StreamContext().Bounds()
-	if bounds == nil {
+	bounds := execute.Bounds{
+		Start: a.StreamContext().Bounds().Start,
+		Stop:  a.StreamContext().Bounds().Stop,
+	}
+	if bounds == (execute.Bounds{}) {
```

Comment:
Does this part need to be changed? I'm a bit uncomfortable with comparing a bounds to the empty struct as a bounds of `[0, 0)` is technically different from nil.

Reply:
I changed it so that the check is performed before the assignment. Does that look better to you?

Reply:
It compares to `nil` now.